test/Serilog.Tests/Capturing/PropertyValueConverterTests.cs

```diff
@@ -396,13 +396,17 @@ public void AllTupleLengthsUpToSevenAreSupportedForCapturing()
             Assert.IsType<SequenceValue>(_converter.CreatePropertyValue(t));
     }
 
+#if !IVALUETUPLE
+
     [Fact]
     public void EightPlusValueTupleElementsAreIgnoredByCapturing()
```

Comment:
@nblumhardt 

i wasnt sure if 8 was a by design choice? or u just didnt want to bloat the conversion code with too many tuple conversions.

if it is by design, let me know and i will make the ITuple respect this

Reply:
Not by design - I think dropping this limitation is good :+1: